superset-frontend/plugins/legacy-plugin-chart-world-map/src/WorldMap.js

```diff
@@ -24,7 +24,7 @@ import {
   getSequentialSchemeRegistry,
   CategoricalColorNamespace,
 } from '@superset-ui/core';
-import Datamap from 'datamaps/dist/datamaps.world.min';
+import Datamap from 'datamaps/dist/datamaps.all.min';
```

Comment:
This could increase the size of our js bundle.

Reply:
yeah main mission is fixing `master` for now ...